Created account page

Joseph-Lintern opened the pull request:
Created account page
Displays user info and hunts they created
Change password form
Modified about_us and privacy_policy
Fixed footer links

Joseph-Lintern committed:
Created account page

Created account page
Displays user info and hunts they created
Change password form
Modified about_us and privacy_policy
Fixed footer links

mf470 committed:
Merge branch 'master' into Add-account-page

mf470 commented:
Passwords do not match doesn't disappear after changing the passwords 
Should use prepared sql statements

mf470 reviewed (changes requested):
Passwords do not match doesn't disappear after changing the passwords
Should use prepared sql statements

Joseph-Lintern committed:
Prepared statements and bug fix

Added prepared statements to both files

Fixed bug so the message doesn't always display
